custom-metrics-stackdriver-adapter/README.md

```diff
@@ -129,18 +151,27 @@ by a simple name, as defined in [custom metric naming rules].
        podID := *podIdFlag
        ```
 
-     - `container_name = ""`
-     - `project_id`, `zone`, `cluster_name` - can be obtained by your application from
-        [metadata server]. You can use Google Cloud compute metadata client to get
-        these values, example in Go:
+     - For monitored resource `gke_container`, `container_name` should be set to
+       `""` to indicate that the metric is associated with a pod, not a
+       particular container.
+     - `project_id`, `zone`, `location`, `cluster_name` - can be obtained by your
```

Comment:
Maybe it worth somehow to split it. 'gke_container' has zone, 'k8s_pod' and other we have location. Also worth to mention that in old resource model we have instance_id (which also can be read from the metadata server).

Reply:
My approach was to don't specify what labels correspond to which resource model and let users find in the linked Stackdriver documentation labels, that they need to set. I want only to focus on how to obtain required label values. But I'm open to suggestions to make it more readable and useful. Project, zone, location and cluster name can be obtained the same way, therefore I group them together. Do you propose to split zone and location to a separate bullet? Or have two bullets with overlapping content: (project, zone, cluster name), (project, location, cluster name)? What about just adding a comment in the code below that zone is used in old model and location - in the new one?

As for instance id, it's mentioned later, as it's not even required for this use case.

Reply:
I would prefer at least specifying that zone is old resource model, location is new.

Reply:
Done